bin/mv.js

```diff
@@ -0,0 +1,128 @@
+var url = require('url');
+var _ = require('lodash');
+var Chalk = require('chalk');
+var Cli = require('structured-cli');
+var ConfigFile = require('../lib/config');
+
+module.exports = Cli.createCommand('mv', {
+    description: 'Move a named webtask',
+    plugins: [
+        require('./_plugins/profile'),
+    ],
+    options: {
+        'target-container': {
+            description: 'Target container',
+            type: 'string',
+            dest: 'targetContainer'
+        },
+        'target-profile': {
+            description: 'Target profile',
+            type: 'string',
+            dest: 'targetProfile'
+        },
+    },
+    params: {
+        'source': {
+            description: 'Webtask name',
+            type: 'string',
+            required: true,
+        },
+        'target': {
+            description: 'Webtask name',
+            type: 'string',
+            required: false,
+        },
+    },
+    handler: handleWebtaskMove,
+});
+
+function handleWebtaskMove(args) {
+    var options = _(args).pick(['targetContainer', 'targetProfile']).omitBy(_.isNull).value();
+    var targetName = args.target || args.source;
+
+    return moveWebtask(args.profile, args.source, {
+        profile: options.targetProfile,
+        container: options.targetContainer,
+        name: targetName
+    }).then(function() {
+        console.log(Chalk.green('Moved webtask: %s'), Chalk.bold(targetName));
+    });
+}
+
+function moveWebtask(profile, name, target) {
+    var source = {
+        name: name,
+        container: profile.container,
+        profile: profile.name
+    };
+
+    if (equal(source, target)) {
+        throw Cli.error.invalid('Webtasks are identical. Use a different target name, container or profile.');
+    }
+
+    return read(profile, name)
+        .then(function(webtask) {
+            return copy(profile, webtask, target);
+        })
+        .then(function() {
+            return profile.removeWebtask({
+                name: name
+            });
+        });
+}
+
+function equal(source, target) {
+    return source.name == target.name && !target.container && !target.profile;
+}
+
+function read(profile, name) {
+    return profile.inspectWebtask({
+            name: name,
+            decrypt: true,
+            fetch_code: true
+        })
+        .catch(function(err) {
+            if (err.statusCode === 404) {
+                throw Cli.error.notFound('No such webtask: ' + Chalk.bold(name));
+            }
+            throw err;
+        })
+        .then(function(webtask) {
```

Comment:
This clause was probably for debugging but no longer does anything and can be removed.